### PR TITLE
Fast Track/Quick Join Functionality

### PR DESCRIPTION
**Pull request by sh1ggy**
Quick join modal functionality

- Componentization of Modal & Join
- pages\[id].tsx for fast track/quick join functionality that copies the peer join functionality
- daisyUI modal component
- Passed UAT using ngrok `ngrok http 3000` -- removed quick-join URL from modal component
- localhost handling with kongroo peerJS server being used now (to pass CORS)

### CODE REVIEW

**Review by ProPablo — CHANGES_REQUESTED** [review]
hey man.

File: custard/pages/_app.tsx [hunk 1]
```diff
@@ -32,10 +32,12 @@ export default function App({ Component, pageProps }: AppProps) {
   const router = useRouter(); // for navigation
 
   useEffect(() => {
-    if (peerRef.current == null && !HomePages.some(h => h == router.pathname)) {
```
Comment (sh1ggy) [inline custard/pages/_app.tsx:35]: thank you.

**Review by ProPablo — COMMENTED** [review]
ebic

File: custard/pages/_app.tsx (hunk 1 above)
Comment (ProPablo) [inline custard/pages/_app.tsx:35]: Nice well done, you should smile NOW

**Review by ProPablo — APPROVED** [review]
Nice bro

File: custard/pages/index.tsx [hunk 2]
```diff
@@ -80,57 +72,60 @@ export default function Home() {
...
   return (
     <>
-      <div className="navbar bg-lavender">
-        <div className="flex-1 flex justify-center mr-auto navbar-center">
-          <button className="btn btn-ghost flex justify-center align-items h-28" onClick={() => router.push("/")}>
+      {/* Fast Track MODAL */}
+      <QuickModal peerId={state.peerId}/>
+
+      {/* TOAST */}
+      <div className={`animate-bounce select-none toast transition-opacity duration-300 text-white z-10000000000 ${idCopy ? 'opacity-100' : 'opacity-0'}`}>
```
Comment (ProPablo) [inline custard/pages/index.tsx]: bruh plz

**Review by ProPablo — CHANGES_REQUESTED** [review]

**Review by ProPablo — APPROVED** [review]
So true bestie

**Pull request merged**